### DISCUSSION TITLE
Since lazy_file got enabled, opening a file with a line number using wsdjeg/vim-fetch now gives errors on startup

### DISCUSSION THREAD

**Discussion by lparry** [LazyVim/LazyVim#1863] [Q&A]
Ever since [this commit](https://github.com/LazyVim/LazyVim/commit/f59cdff4d9f04ff349cefdee0b363d573954ce0d) I've been seeing errors on startup any time I open a file with a line number (using [vim-fetch](https://github.com/wsdjeg/vim-fetch))

eg
```
$ nvim path/to/some_file.rb:20
```

```
Error executing vim.schedule lua callback: ...ocal/share/nvim/lazy/LazyVim/lua/lazyvim/util/plugin.lua:99: Invalid buffer id: 1
stack traceback:
        [C]: in function '__index'
        ...ocal/share/nvim/lazy/LazyVim/lua/lazyvim/util/plugin.lua:99: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
Press ENTER or type command to continue
Error executing vim.schedule lua callback: ...ocal/share/nvim/lazy/LazyVim/lua/lazyvim/util/plugin.lua:83: Vim:E367: No such group: "lazy_file"
stack traceback:
        [C]: in function 'nvim_del_augroup_by_name'
        ...ocal/share/nvim/lazy/LazyVim/lua/lazyvim/util/plugin.lua:83: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

The correct file is opened at the correct line, but the errors are kind of annoying.

Can someone please fill me on what I need to change to make this error go away? Or even just how to disable lazy_file in config instead of modify the lazyvim source in `~/.local`?

**Comment by folke** [comment]
Not sure how that can happen tbh, but fixed it anyway :)

**Comment by lparry** [comment]
Thanks @folke, that has sorted out the error at `LazyVim/lua/lazyvim/util/plugin.lua:83`, but now I see two occurrences of the error on line 99 of the same file

```

Error executing vim.schedule lua callback: ...ocal/share/nvim/lazy/LazyVim/lua/lazyvim/util/plugin.lua:99: Invalid buffer id: 1
stack traceback:
        [C]: in function '__index'
        ...ocal/share/nvim/lazy/LazyVim/lua/lazyvim/util/plugin.lua:99: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
Press ENTER or type command to continue
Error executing vim.schedule lua callback: ...ocal/share/nvim/lazy/LazyVim/lua/lazyvim/util/plugin.lua:99: Invalid buffer id: 1
stack traceback:
        [C]: in function '__index'
        ...ocal/share/nvim/lazy/LazyVim/lua/lazyvim/util/plugin.lua:99: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
Press ENTER or type command to continue
```

I don't really know lua or the neovim apis very well, but following your example I (and assuming pcall swallows exceptions) I found that replacing line 99 with

```diff
-      if vim.bo[event.buf].filetype then
+      if pcall(vim.bo[event.buf], "filetype") then
```

I no longer see any error, but I'm not sure what the behaviour of `lazy_file` actually is to know if that changes breaks it 🤷‍♂️
